charts/leonardo/templates/roleBinding.yaml

```diff
@@ -1,12 +1,12 @@
 apiVersion: rbac.authorization.k8s.io/v1
 kind: RoleBinding
 metadata:
-  name: leonardo-sa-binding
+  name: leonardo-ksa-binding
```

Comment:
Should this be `leonardo-sa` / `leonardo-sa-binding` to stick with conventions in other charts? Especially since the custom psp is no longer needed

Reply:
In leonardo team, we refer to kubernetes sa as ksa, and normal google service account as sa...I think it's confusing when we use sa here, because most people know it as google service account instead of ksa. And in the chart, we do refer to leonardo's app service account as Leonardo sa.

Reply:
yeah, I hear that... But I think it's valuable to have things follow a common standard. It'll be a bit awkward if we have `sam-sa`, `rawls-sa`, `cromwell-sa`, and then, randomly, `leonardo-ksa`... As a DevOps engineer inspecting a Terra Kubernetes namespace, I'd be wondering if `leonardo-ksa` is intended for a different purpose than these other application SAs

Reply:
reverted